orchagent/aclorch.h

```diff
@@ -430,13 +433,21 @@ class AclOrch : public Orch, public Observer
     static void collectCountersThread(AclOrch *pAclOrch);
 
     bool createBindAclTable(AclTable &aclTable, sai_object_id_t &table_oid);
-    sai_status_t bindAclTable(sai_object_id_t table_oid, AclTable &aclTable, bool bind = true);
+    sai_status_t bindAclTable(AclTable &aclTable, bool bind = true);
     sai_status_t deleteUnbindAclTable(sai_object_id_t table_oid);
 
+    bool isAclTableTypeUpdated(acl_table_type_t table_type, AclTable &aclTable);
     bool processAclTableType(string type, acl_table_type_t &table_type);
+    bool isAclTableStageUpdated(acl_stage_type_t acl_stage, AclTable &aclTable);
     bool processAclTableStage(string stage, acl_stage_type_t &acl_stage);
+    bool isAclTablePortListUpdated(string portList, AclTable &aclTable);
     bool processAclTablePorts(string portList, AclTable &aclTable);
     bool validateAclTable(AclTable &aclTable);
+    bool updateAclTablePorts(AclTable &newTable, AclTable &curTable);
+    void getAddDeletePorts(AclTable    &newT,
+		           AclTable    &curT,
+			   set<string> &addSet,
+			   set<string> &delSet);
```

Comment:
Can you add description for the reasons for the changes?  
And where are the changes in the .cpp file?